library/classes/Document.class.php

```diff
@@ -762,4 +762,13 @@ function postPatientNote($provider, $category_id, $message = '')
         // Link the new note to the document.
         setGpRelation(1, $this->get_id(), 6, $noteid);
     }
+      
+  /**
+   * Return note objects associated with this document using Note::notes_factory
+   *
+   */
+    function get_notes()
+    {
+        return (Note::notes_factory($this->get_id()));
```

Comment:
Did not want to confuse since this PR does not deal with Note class itself. notes_factory should be changed to `public static` - which is how it is coded and used currently.  
We could create new Note() but that will continue to mislead subsequent developers.